universal-login-react/src/ui/Onboarding/EmailFlowChooser.tsx

```diff
@@ -0,0 +1,65 @@
+import React, {useState} from 'react';
+import {useClassFor, classForComponent} from '../utils/classFor';
+import '../styles/base/onboardingSelectFlow.sass';
+import '../styles/themes/UniLogin/onboardingSelectFlowThemeUniLogin.sass';
+import Input from '../commons/Input';
+import {Label} from '../commons/Form/Label';
+
+type EmailFlow = {
+  kind: 'create',
+  onClick: () => void;
+} | {
+  kind: 'connect',
+  onClick: () => void,
+}
+
+interface EmailFlowChooserProps {
+  onCreateClick: () => void;
+  onConnectClick: () => void;
+}
+
+export const EmailFlowChooser = ({onCreateClick, onConnectClick}: EmailFlowChooserProps) => {
+  const [flow, setFlow] = useState<EmailFlow>({kind: 'create', onClick: onCreateClick});
+
+  return (
+    <div className={useClassFor('select-flow')}>
+      <div className={useClassFor('flow-wrapper')}>
+        <div className={useClassFor('user-tabs')}>
+          <button
+            onClick={() => setFlow({kind: 'create', onClick: onCreateClick})}
+            className={`${classForComponent('user-tab')} ${flow.kind === 'create' ? 'active' : ''}`}>New user</button>
+          <button
+            onClick={() => setFlow({kind: 'connect', onClick: onConnectClick})}
+            className={`${classForComponent('user-tab')} ${flow.kind === 'connect' ? 'active' : ''}`}>Existing user</button>
+        </div>
+        <div className={useClassFor('flow-content')}>
+          {flow.kind === 'create' && <CreationContent/>}
+          {flow.kind === 'connect' && <ConnectionContent/>}
+        </div>
+      </div>
+      <button onClick={flow.onClick} className={classForComponent('confirm-btn')}>Confirm</button>
+    </div>
+  );
+}
+
+const CreationContent = () => {
```

Comment:
This component should have some internal state e.g. ensName or e-mail that will change in the input.

Reply:
The same with ConnectionContent